run_scans.py

```diff
@@ -215,26 +245,34 @@ def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nev
                          [options.throttle for x in range(len(chamber_config))],
                          [options.internal for x in range(len(chamber_config))],
                          [options.debug for x in range(len(chamber_config))]
+                         [options.calSF for x in range(len(chamber_config))]
+                         [options.chMin for x in range(len(chamber_config))]
+                         [options.chMax for x in range(len(chamber_config))]
+                         [options.pDel for x in range(len(chamber_config))]
+                         [options.voltageStepPulse for x in range(len(chamber_config))]
+                         [options.latency for x in range(len(chamber_config))]
+                         [options.CalPhase for x in range(len(chamber_config))]
+
```

Comment:
you're missing comma's after each entry here and you have not closed the initial parentheses.

Reply:
this is most likely where the build error is coming from